php/libraries/Utility.class.inc

```diff
@@ -587,54 +587,6 @@ class Utility
         return $instruments;
     }
 
-    /**
-     * Get a list of all instruments which have scoring implemented.
-     * Used in statistics module.
-     *
-     * @return array non-associative array of test_names
-     */
-    static function getScoredInstruments(): array
```

Comment:
This function is never used in LORIS.

Reply:
it used to be used by the statistics module (and DQT?) in order to get a list of instruments and there columns that should be displayed for scatterplots. I don't think it's used anymore.